db/migrate/20230501194608_convert_enclosures.rb

```diff
@@ -0,0 +1,31 @@
+class ConvertEnclosures < ActiveRecord::Migration[7.0]
+  def up
+    add_column :episodes, :medium, :integer
```

Comment:
what makes medium type integer? an enum?

Reply:
Yep, it's an enum of `audio` / `video`.  Though I did think of just making a string.  But the other enums in Feeder seem to be integer fields?

I'm still a bit conflicted on this one.  There's also a `media_resources.medium` string field.  But that field is more free-form, since you could theoretically upload any file type.  (We only validate the media_resource medium is audio/video when status == complete).